client/modules/dam/src/views/asset/modals/edit.js

```diff
@@ -69,21 +69,21 @@ Espo.define('dam:views/asset/modals/edit', 'views/modals/edit',
 
         actionSimpleUpload() {
             this.actionClose();
-            this.actionQuickCreate('detailSmall');
+            this.actionQuickCreate(false);
         },
 
         actionMassUpload() {
             this.actionClose();
-            this.actionQuickCreate('massCreateDetailSmall');
+            this.actionQuickCreate(true);
         },
 
-        actionQuickCreate(layout) {
+        actionQuickCreate(massCreate) {
             let options = _.extend({
                 model: this.model,
                 scope: this.scope,
-                attributes: {}
+                attributes: {massCreate}
```

Comment:
Masscreate is a bool variable. So how it should work here?

Reply:
it is a bool variable, if it is true, the files field will be shown in the layout and the file and name will not be shown